python/cugraph/cugraph/experimental/__init__.py

```diff
@@ -73,3 +73,7 @@
 
 overlap = experimental_warning_wrapper(EXPERIMENTAL__overlap)
 overlap_coefficient = experimental_warning_wrapper(EXPERIMENTAL__overlap_coefficient)
+
+from cugraph.gnn.data_loading import EXPERIMENTAL__BulkSampler
+
+BulkSampler = experimental_warning_wrapper(EXPERIMENTAL__BulkSampler)
```

Comment:
I'm not too worried about it, but this will lose the namespace (ie. it will be `cugraph.experimental.BulkSampler` instead of `cugraph.experimental.gnn.BulkSampler` assuming we want the latter).  The (undocumented, which is my fault) convention is that a user can simply remove `.experimental` from their imports and nothing else once it's promoted.

Reply:
moved